atom/common/api/api_messages.h

```diff
@@ -37,6 +38,20 @@ IPC_MESSAGE_ROUTED3(AtomViewMsg_Message,
 
 IPC_MESSAGE_ROUTED0(AtomViewMsg_Offscreen)
 
+IPC_MESSAGE_ROUTED4(AtomAutofillViewMsg_ShowPopup,
```

Comment:
I have used the chromium convention instead, using XXXFrameHost... / XXXFrame... for messages sent between `RenderFrame` and `RenderFrameHost`.